### DISCUSSION TITLE
Non-relative module imports

### DISCUSSION THREAD

**Discussion by emersonlaurentino** [vtex/faststore#902] [Ideas] [answered]
It would be nice to be able to import without having the relative import path.

## Exemple

Currently, in the [Bullets](https://github.com/vtex/faststore/blob/master/packages/store-ui/src/molecules/Bullets/Bullets.tsx) component, we import the Button component.

So the import is:
```tsx
import Button from '../../atoms/Button'
```

With Non-relative import, we would have something like this:

```tsx
import Button from 'atoms/Button'
```

To understand how to implement and what the implications are, [here is the official Typescript documentation](https://www.typescriptlang.org/docs/handbook/module-resolution.html#relative-vs-non-relative-module-imports).

We can just start in the `store-ui` package and add this setting to the other packages too, what do you think?

**Comment by tlgimenes** [comment, marked as answer]
This would be awesome! please! Also, store-sdk should be really simple to do this too. A nice first issue!

**Comment by emersonlaurentino** [comment]
okay, on store-ui i implemented on #912

**Comment by victorhmp** [comment]
@emersonlaurentino could you make it more clear what the advantages of making this change are?
The docs explain how this works, but I failed to see a clear advantage over the relative imports we currently use.

Also, wouldn't this be easier to do once we have a more stable structure in place for all components from `store-ui`? Because we might still make changes to where a component is located.

**Reply by emersonlaurentino** [reply]
Great questioning! There are some problems in some IDE's to autocomplete and there's the well-known `SlashPathHell`: ../../../path/to/file.js, where we put several "../" to lower or level up the directories.
